README.adoc

```diff
@@ -158,32 +197,52 @@ userRegistry.xml
 
-Map the groups in the user registry to the appropriate user roles supported by the application for proper user authorization. The [hotspot=group-name-Manager file=1]`Manager` and [hotspot=group-name-TeamLead file=1]`TeamLead` groups are mapped to the [hotspot=security-role-admin file=1]`admin` role while the [hotspot=group-name-Employee file=1]`Employee` group is mapped to the [hotspot=security-role-user file=1]`user` role using the [hotspot=Security file=1]`<security-role/>` and [hotspot=Group file=1]`<group/>` elements under the [hotspot=application-bnd file=1]`<application-bnd/>` configuration.
+The [hotspot file=1]`server.xml` file contains the security configuration of the server 
+under the [hotspot=application-bnd file=1]`<application-bnd/>` element.
+Map the groups in the [hotspot file=0]`userRegistry.xml` to the appropriate user roles supported by the application for proper user authorization,
+using the [hotspot=Security file=1]`<security-role/>` and 
+[hotspot=Group hotspot=group-name-Employee file=1]`<group/>` elements. 
+The [hotspot=group-name-Manager file=1]`Manager` and [hotspot=group-name-TeamLead file=1]`TeamLead` groups are mapped to the 
+[hotspot=security-role-admin file=1]`admin` role while the [hotspot=group-name-Employee file=1]`Employee` group is mapped to the 
+[hotspot=security-role-user file=1]`user` role.
 
 
 // =================================================================================================
-// Building and running the application
+// Running the application
 // =================================================================================================
 [role="command"]
-include::{common-includes}/mvnbuild.adoc[]
+include::{common-includes}/devmode-build.adoc[]
 
-Point your browser to http://localhost:9080[http://localhost:9080^]. As you can see, the browser gets redirected from an HTTP connection to an HTTPS connection automatically, which is due to the transport guarantee defined in the `HomeServlet` class.
+HomeServlet.java
+[source, java, linenums, role='code_column hide_tags=copyright,javaDoc1,javaDoc2']
+----
+include::finish/src/main/java/io/openliberty/guides/ui/HomeServlet.java[]
+----
+
+Point your browser to http://localhost:9080[http://localhost:9080^]. 
+As you can see, the browser gets redirected from an HTTP connection to an HTTPS connection automatically, 
+which is due to the transport guarantee defined in the [hotspot file=0]`HomeServlet` class.
 
-You will see a login form since form authentication is implemented and configured. Sign into the application using one of the credentials from the following table, which is defined in the configured user registry.
+You will see a login form since form authentication is implemented and configured. 
+Sign into the application using one of the credentials from the following table, which is defined in the configured user registry.
```

Comment:
Update to:
"Sign in to the application by using one of the credentials from the following table. The credentials are defined in the configured user registry."